sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -17,6 +17,9 @@ public abstract partial class AsyncPageable<T> : System.Collections.Generic.IAsy
     }
     public static partial class AzureCoreExtensions
     {
+        public static dynamic ToDynamicFromJson(this System.BinaryData data) { throw null; }
```

Comment:
I wonder if this parameter should not be called "utf8Json" to hint that the data has to be in a specific format.

Reply:
I'm not sure you will see the parameter name in the extension method, but happy to make the change for the docs/anyone who reads the source directly.